### ISSUE TITLE
Card Length Incorrect: 0

### ISSUE THREAD

**Issue by chrislatimer** [bupticybee/TexasSolver#130]
<img width="1046" alt="image" src="https://user-images.githubusercontent.com/831081/191824871-cf02c9e5-d48b-4b9a-be68-da285ef0defc.png">

Running current distribution for MacOS, just trying to run with the default ranges/board and seeing this error? Is there something else I need to configure or is this a bug?

**Comment by bupticybee** [comment]
> <img alt="image" width="1045" src="https://user-images.githubusercontent.com/831081/191853209-d019ffea-03c9-4092-9bbf-2ba215b284ce.png">
> 
> Running current distribution for MacOS, just trying to run with the default ranges/board and seeing this error? Is there something else I need to configure or is this a bug?

It happens sometime in mac, Havn't been able to locate and solve this problem. But this should not happen everytime.

**Comment by maosatgithub** [comment]
I think this happens, when the "Build Tree" was not done before "Start Solving", independent of OS.

**Comment by bupticybee** [comment]
> I think this happens, when the "Build Tree" was not done before "Start Solving", independent of OS.

Thank you for clearification.

**Comment by DarthNZ** [comment]
Yup. Fixed the Problem. Darth Vader, FB, Auckland, New Zealand. 21:50 A28 Oct 2024 (UTC).
